source/agora/test/SlashingMisbehavingValidator.d

```diff
@@ -154,6 +154,9 @@ unittest
     utxo_set.peekUTXO(frozen_hash, utxo);
     assert(utxo.type == TxType.Freeze);
 
+    // node[5] becomes unresponsive in order not to slash itself.
```

Comment:
I'm not sure I follow this comment. Will the unittest crash if the sleep isn't here ?

Reply:
Yes, it will because I have added the assertion in case of slashing itself.
The statement makes the `node[5]` not to nominating a consensus data that contains the `missing_validators` having the index of the node itself.

Reply:
So the assertion *does* trigger. It should never trigger. When we discussed this, I suggested to add the assertion if the bug was fixed, but it seems it is not.

Reply:
Sorry for the misunderstanding. I will add the code that does not make a consensus data if a node itself doesn't reveal its pre-images.

Reply:
Removed